test/bricks-net-url.test.cc

```diff
@@ -0,0 +1,261 @@
+/*******************************************************************************
```

Comment:
Ah, got it, sweet. Still, easy to get out of sync. `TODO(sompylasar+dkorolev)`: Think of a better way.

(Perhaps a default `make` target in Bricks to re-generate a JS test?)